src/services/print.js

```diff
@@ -488,6 +492,49 @@ ngeo.Print.prototype.encodeVectorStyleStroke_ =
 
 
 /**
+ * @param {Array.<MapFishPrintSymbolizer>} symbolizers Array of MapFish Print
+ *     symbolizers.
+ * @param {!ol.style.Text} textStyle Text style.
+ * @private
+ */
+ngeo.Print.prototype.encodeTextStyle_ = function(symbolizers, textStyle) {
+  var symbolizer = {
+    type: 'Text'
+  };
```

Comment:
Look at the [MapFishPrintSymbolizerPolygon type](https://github.com/camptocamp/ngeo/blob/master/externs/mapfish-print-v3.js#L428-L462) as an example. You'd need something similar for Text.